constants.xml

```diff
@@ -210,6 +214,8 @@
         </peer>
     -->
     </lookups>
+    <upper_seed/>
+    <lower_seed/>
```

Comment:
Supposedly, the naming rule inside constant file should be \<sectionName\>\<TAG1\>...\</TAG1\>\<TAG2/\>...\</sectionName\>, is it possible to make it consistent?

Reply:
these would contain Ips. as you can see lookup ip is also outside every scope